src/functions/searchResults/framework/handler.ts

```diff
@@ -102,8 +102,9 @@ export async function handler(event: APIGatewayEvent, fnCtx: Context): Promise<R
       queryParameters.staffNumber = staffNumber;
     }
 
-    const result = await new SearchRepository().searchForTestResultWithDriverDetails(queryParameters);
-
+    const result = await getConciseSearchResults(queryParameters);
+    console.log('The result is');
```

Comment:
Getting undefined here, even though I'm mocking and returning a promise that should return a TestResultRecord